Subject: [xf86-video-dummy] make packet sysroot aware

The xf86-video-dummy packet tries to include headers from
$(prefix)/include, which isn't correct. Usually, when building
distributions, packets are configured with --prefix=/usr and installed
with SYSROOTS=....

Signed-off-by: Luotao Fu <l.fu@pengutronix.de>
Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: xf86-video-dummy-0.3.2/configure.ac
===================================================================
--- xf86-video-dummy-0.3.2.orig/configure.ac
+++ xf86-video-dummy-0.3.2/configure.ac
@@ -58,7 +58,7 @@ PKG_CHECK_MODULES(XORG, [xorg-server >= 
 sdkdir=$(pkg-config --variable=sdkdir xorg-server)
 
 CFLAGS="$CFLAGS $XORG_CFLAGS "' -I$(top_srcdir)/src'
-INCLUDES="$XORG_INCS -I${sdkdir} "'-I$(top_srcdir)/src -I$(prefix)/include'
+INCLUDES="$XORG_INCS -I${sdkdir} "'-I$(top_srcdir)/src '"-I${SYSROOT}"'$(prefix)/include'
 AC_SUBST([CFLAGS])
 AC_SUBST([INCLUDES])
 
